### PR TITLE
Theme Submission: Gastsby Simple Blog

### PR DESCRIPTION
**Pull request by collinsmuriuki**

### CODE REVIEW

File: content/theme/gatsby-starter-simple.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Gastsby Simple Blog"
+github: https://github.com/thecodemint/gatsby-starter-simple
+demo: https://thecodemint.github.io/gatsby-starter-simple/
+author: collinsmuriuki
+date: 2020-04-14
+github_branch: master
+ssg:
```
Comment (JugglerX) [inline content/theme/gatsby-starter-simple.md:8]: Please add above ssg
```
cms:
  - No Cms
```

**Comment by collinsmuriuki** [comment]
Done

File: content/theme/gatsby-starter-simple.md (hunk 1 above)
Comment (collinsmuriuki) [reply content/theme/gatsby-starter-simple.md:8]: Fixed

**Comment by JugglerX** [comment]
Sorry another thing holding this up. We need to add disquis and analytics as services. This is done by creating content, icons etc in the content directory. Take a look at the services folder.

It will be a few more days before I can get to this. Sorry for the delay

**Comment by collinsmuriuki** [comment]
I have left the services out to keep things simple if you don't mind

**Comment by JugglerX** [comment]
No problem merged then

**Pull request merged**